### ISSUE TITLE
Normlizar datos

### ISSUE THREAD

**Issue by ghost** [Exploratorio-DCC-PUC/Syllabus#87]
¿Podemos utilizar la librería Scikit-learn (en particular, importar preprocessing)  para normalizar las columnas de los datos?. Pregunto porque utilicé también numpy para hacerlo pero al parecer lo que hace es normalizar tanto filas como columnas, por lo que arroja resultados un poco extraños.

**Comment by Mdelaf** [comment]
Hola Felipe,

Sí, sí pueden. Pueden usar `sklearn`, `numpy`, `pandas` y `altair` para lo que quieran.
